.circleci/config.yml

```diff
@@ -0,0 +1,30 @@
+version: 2
+jobs:
+  build-and-test:
+    working_directory: ~/ITKAnisotropicDiffusionLBR-build
+    docker:
+      - image: insighttoolkit/module-ci:latest
+    steps:
+      - checkout:
+          path: ~/ITKAnisotropicDiffusionLBR
+      - run:
+          name: Configure
+          command: |
+            ls ~/
+            cmake \
+              -G Ninja \
+              -DITK_DIR:PATH=/ITK-build \
+              -DBUILD_TESTING:BOOL=ON \
+              -DBUILDNAME:STRING=External-ITKAnisotropicDiffusionLBR-${CIRCLE_BRANCH}-${CIRCLE_BUILD_NUM} \
+                ~/ITKAnisotropicDiffusionLBR
+      - run:
+          name: Build and Test
+          command: |
+            ctest -VV -D Experimental
+
+workflows:
+    version: 2
+    build-test-package:
```

Comment:
Look like the packaging step is missing ?

Reply:
Should have marked it as WIP -- should be about ready, now.